src/dev/flang/tools/docs/Html.java

```diff
@@ -137,6 +137,38 @@ private String summary(AbstractFeature af)
       + "</div>";
   }
 
+  /**
+   * list of features that are redefined by feature af
+   * @param af
+   * @return list of redefined features, as HTML
+   */
+  private String redefines(AbstractFeature af, boolean recurse)
```

Comment:
I had trouble understanding the `recurse` option. If you split this up into a recursive part and one that wraps the result in the `div>ul` then it should be easier to read. I also think the `af.redefines().isEmpty()` should not be necessary?